tedana/decay.py

```diff
@@ -2,10 +2,26 @@
 Functions to estimate S0 and T2* from multi-echo data.
 """
 import numpy as np
+import scipy
 from tedana import utils
 
+def mono_exp(tes, s0, t2star):
+    """
+    Specifies a monoexponential model for fitting
+
+    Parameters
+    ----------
+    tes : (E,) :obj:`list`
+        Echo times
+    s0 : :obj:`float`
+        Initial signal parameter
+    t2star : :oj:`float`
+        T2* parameter  
+
+    """
+    return s0 * np.exp(-tes/t2star)
 
-def fit_decay(data, tes, mask, masksum):
+def fit_decay(data, tes, mask, masksum, fittype):
```

Comment:
I think to fix the unit test failures you'll need to change it as follows:
```suggestion
def fit_decay(data, tes, mask, masksum, fittype='loglin'):
```

Reply:
Should I do that, or change the tests themselves? I'm still quite vague/confused on the subject of testing so trying to get a feel for best practices...

Reply:
IMO we should use default parameters for this addition, since adding a new parameter is a breaking change for `fit_decay` and we're going to recommend log-linear by default. @emdupre and @tsalo may have differing opinions on this though.

Reply:
I just added in the defaults to the tests - and that worked nicely. After I fixed the commas (twice...).

Reply:
added the default to testing - this seems  to have resolved the issues.